opt/se-logger/liveupdate.py

```diff
@@ -418,7 +420,7 @@ def parse0500(data):
   while pos < len(data):
     type, id, length, timestamp = struct.unpack("<HLHL", data[pos:pos+12])
     if type == 0x0080 and length == 13:  # Optimizer data (packed)
-      bytes = map(ord, data[pos+12:pos+8+length])
+      bytes = list(map(lambda x: ord(chr(x)), data[pos+12:pos+8+length]))
```

Comment:
```suggestion
      bytes = data[pos + 12 : pos + 8 + length]
```

Reply:
Thx! Since the ord(chr()) construct is not needed, this simplification makes much more sense.